epaint/src/text/text_layout_types.rs

```diff
@@ -257,6 +257,54 @@ impl TextFormat {
 
 // ----------------------------------------------------------------------------
 
+#[derive(Clone, Debug, PartialEq)]
+#[cfg_attr(feature = "serde", derive(serde::Deserialize, serde::Serialize))]
+pub struct TextWrapping {
+    /// Try to break text so that no row is wider than this.
+    /// Set to [`f32::INFINITY`] to turn off wrapping.
+    /// Note that `\n` always produces a new line.
+    pub max_width: f32,
+
+    /// Maximum amount of rows the text should have.
+    /// Set to `0` to disable this.
+    pub max_lines: usize,
```

Comment:
I think `usize::MAX` is a better way to disable it; then there is no need for any special cases.

Reply:
Default was initially `usize::MAX`, but I then later realized that if it's set to `0`, then egui crashes in `rows_from_paragraphs` because we end up with no paragraphs, hence the check in `line_break` function.